disco/api/http.py

```diff
@@ -264,11 +263,11 @@ def call(self, route, args=None, **kwargs):
         # Possibly wait if we're rate limited
         response.rate_limited_duration = self.limiter.check(bucket)
 
-        self.log.debug('KW: %s', kwargs)
```

Comment:
All these logging format strings are correct. These are not strictly Python 2 format strings but are instead formatted internally by the logging package.